website/www/site/assets/js/quotes-slider.js

```diff
@@ -10,33 +10,99 @@
 // License for the specific language governing permissions and limitations under
 // the License.
 
-var slider = new KeenSlider("#my-keen-slider", {
-  loop: true,
-  created: function (instance) {
-    var dots_wrapper = document.getElementById("dots");
-    var slides = document.querySelectorAll(".keen-slider__slide");
-    slides.forEach(function (t, idx) {
-      var dot = document.createElement("button");
-      dot.classList.add("dot");
-      dots_wrapper.appendChild(dot);
-      dot.addEventListener("click", function () {
-        instance.moveToSlide(idx);
-      });
+function calcBodyWidth() {
+  return window.innerWidth && document.documentElement.clientWidth ?
+    Math.min(window.innerWidth, document.documentElement.clientWidth) :
+    window.innerWidth ||
+    document.documentElement.clientWidth ||
+    document.getElementsByTagName('body')[0].clientWidth;
+}
+
+(function () {
+
+  var CountOfSlides = {
+    min: 1,
+    renderedOnDesktop: 3,
+  }
+
+  var Selectors = {
+    desktopSlider: '.quotes-desktop.keen-slider-JS',
+    mobileSlider: '.quote-mobile.keen-slider-JS',
+    oneSlide: '.keen-slider__slide',
+  }
+
+  var classOneSlideExtra = 'wrap-slide';
+  var classOneSlide = 'keen-slider__slide';
+  var classVisible = 'visible';
+
+  var tabletWidth = 1024;
+  var bodyWidth = calcBodyWidth();
+  var isDesktopWidth = bodyWidth >= tabletWidth;
+
+  var currentSliderSelector = isDesktopWidth ? Selectors.desktopSlider : Selectors.mobileSlider;
+  var currentSliderDOM = document.querySelector(currentSliderSelector);
+  currentSliderDOM.classList.add(classVisible);
+
+  var slidesDOM = currentSliderDOM.querySelectorAll(".keen-slider__slide");
+  var actualCountOfSlides = currentSliderDOM.querySelectorAll(Selectors.oneSlide).length;
+
+  // create and add additional div wrappers over group of 3 cards
+  if (isDesktopWidth) {
```

Comment:
Is it possible to move the contents of this if statement into a separate function? It would decrease the size of the parent function and make the comment unnecessary

Reply:
done